spec/yardstick/v2_client/grant_status_spec.rb

```diff
@@ -0,0 +1,26 @@
+require 'spec_helper'
+
+describe Yardstick::V2Client::GrantStatus do
+  let(:gs_client) { Yardstick::V2Client::GrantStatus }
+  let(:token) { 'abssclklkjlkj' }
+
+  describe :find do
+    it 'should populate the token as part of the response handling' do
+      stub_path_response(:get, '/v2/grant_status/100', 200, { id: 100 }, token: token)
```

Comment:
Line is too long. [86/80]<br>Prefer double-quoted strings unless you need single quotes to avoid extra backslashes for escaping.